docs/developers-guide.md

```diff
@@ -120,7 +121,7 @@ All frontend tests are located in `frontend/test` directory. Run all frontend te
 yarn test
 ```
 
-which will run unit, integration and Cypress end-to-end tests in sequence.
+which will run unit, timezones and Cypress end-to-end tests in sequence.
```

Comment:
Maybe I should've omitted `timezones`? After all, they are a form of unit tests if I'm not mistaken.

Reply:
Yes, it is a form of unit tests (although I'm not sure why there is a separate Jest config for that, though)

Reply:
We run those with different pairs of client/reporting timezones. In the context of unit tests, the "client" timezone is node's timezone.

There have been bugs caused by the browser being east or west of the MB reporting timezone.

[Here's where we run those.](https://github.com/metabase/metabase/blob/master/frontend/test/__runner__/run_timezone_tests)

Reply:
So, the final verdict is...? :)
Leave that piece of text or remove it? Not that it makes much difference, but I'm asking simply for the "correctness" sake.

Reply:
Ah sorry I didn't comment on that. I might drop it, but either way is fine.

Reply:
Dropped, done :)
Thanks